stories/Tables/Tables.stories.tsx

```diff
@@ -0,0 +1,641 @@
+import React from 'react';
+
+export const TableBase = () => (
+  <table className='table'>
```

Comment:
I suggest to use the exported `Table` component here, which alreeady includes some props for styling

Reply:
Fixed